lambdas/service/app.py

```diff
@@ -184,13 +184,22 @@ def openapi():
 @app.route('/health', methods=['GET'], cors=True, spec={
     'parameters': [
         {
-            'in': 'query',
-            'name': 'foo',
-            'description': 'This is not a real parameter',
-            'required': False,
-            'schema': {'$ref': "#/components/schemas/Empty"}
+            'name': 'keys',
+            'in': 'path',
+            'required': True,
+            'schema': {'type': 'string'}
```

Comment:
This is specifying that the `/health` endpoint has a required path parameter `keys`. This is incorrect, no?
`/health/{keys}` has a required path parameter but `/health` does not.